src/app/core/services/rundown-timing-context-state.service.ts

```diff
@@ -0,0 +1,187 @@
+import { Rundown } from '../models/rundown'
+import { BehaviorSubject, Observable, Subscription, SubscriptionLike } from 'rxjs'
+import { RundownStateService } from './rundown-state.service'
+import { RundownTimingService } from './rundown-timing.service'
+import { RundownTimingContext } from '../models/rundown-timing-context'
+import { Injectable } from '@angular/core'
+import { Segment } from '../models/segment'
+
+const HIGH_RESOLUTION_INTERVAL_DURATION_IN_MS: number = Math.floor(1000 / 30)
+const LOW_RESOLUTION_INTERVAL_DURATION_IN_MS: number = Math.floor(1000 / 4)
+
+@Injectable()
+export class RundownTimingContextStateService {
+  private timeResolutionTimerId?: ReturnType<typeof setTimeout>
```

Comment:
I was a bit torn between these two. Personally I like Node.Timeout since it is the type. But the type has nothing to do with Node, since this is a browser based project.

Reply:
But isn't it technically still the type?